README.md

```diff
@@ -139,6 +139,36 @@ The application will automatically register with Spring Boot Admin upon start up
 
 Log in to the Spring Boot Admin UI at `http://localhost:8080` to interact with the application. 
 
+### aiohttp
+The following example is complete and should run as is.
+
+```python
+from aiohttp import web
+from pyctuator.pyctuator import Pyctuator
+
+app = web.Application()
+routes = web.RouteTableDef()
+
+@route.get("/")
```

Comment:
should this be `@routes.get("/")`?

Reply:
True (and IntelliJ showed a warning... cool)